docs/indexing.md

```diff
@@ -131,7 +131,7 @@ This step takes around 10 minutes. It will create a `theses.json` file in `home/
 Before running this task, turn off the sneakers workers on production so that any updates that come through after the full index has been generated will be included in the new index and not lost when you swap out the old index.
 
 To turn off sneakers workers:
-- cd in your local princeton_ansible directory -> pipenv shell -> ansible orangelight_alma_prod -u pulsys -m shell -a "sudo service orangelight-sneakers stop"
+- cd in your local princeton_ansible directory -> pipenv shell -> ansible orangelight_prod -u pulsys -m shell -a "sudo service orangelight-sneakers stop"
 
```

Comment:
the name is `orangelight_production` https://github.com/pulibrary/princeton_ansible/blob/2949e896c6f2f8685434dcfe2444092bd00f9149/hosts#L110